### PR TITLE
Add Win_x86/Win_X64 builds and improve JS code

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,71 +64,195 @@
...
+								{
+										keys.push(key);
+								}
+						}
+						return keys;
+				}
+
+				function format_size(bytes) {
+					var i = 0;
+					var units = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```
Comment (skmp) [inline index.html:113]: How about power of 2\* units? KiB, etc?

**Comment by skmp** [comment]
I generally find underscore code much cleaner than raw javascript -- and performance is never a consideration on the web wasteland of cpu cycles. Any reason you don't like it?

**Comment by skmp** [comment]
I must say the end result looks BEAUTIFUL though :D

File: index.html (hunk 1 above)
Comment (Holzhaus) [inline index.html:113]: We could, but I thought that KB is more known than KiB.

Comment (skmp) [inline index.html:113]: All non-apple users use KiB (though, it's written as KB). Afaik apple
started using 1000 based units in 2006 or so.

On Wed, Aug 26, 2015 at 7:28 PM, Jan Holthuis notifications@github.com
wrote:

> In index.html
> https://github.com/reicast/reicast-builds/pull/3#discussion_r38009880:
> 
> > -               function keys(obj)
> > -               {
> > -                       var keys = [];
> > -                       for(var key in obj)
> > -                       {
> > -                               if(obj.hasOwnProperty(key))
> > -                               {
> > -                                       keys.push(key);
> > -                               }
> > -                       }
> > -                       return keys;
> > -               }
> >   +
> > -               function format_size(bytes) {
> > -                   var i = 0;
> > -                   var units = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
> 
> We could, but I thought that KB is more known than KiB.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/reicast/reicast-builds/pull/3/files#r38009880.

## 

~skmp

Comment (Holzhaus) [inline index.html:113]: Yeah, I'll change to KiB (Using Kilo for 1024 is just plain wrong ;-))

Comment (Holzhaus) [inline index.html:113]: By the way, Nautilus on Linux (GNOME 3) seems to use KB = 10^3 bytes rather than K(i)B = 2^10 B.

**Pull request merged**